bedrock/settings/base.py

```diff
@@ -1121,6 +1121,7 @@ def lazy_email_backend():
     '*.tiles.mapbox.com',
     'api.mapbox.com',
     'creativecommons.org',
+    'servedby.flashtalking.com',
```

Comment:
I'd like to put this behind the switch as well. We can't import the `switch()` helper here, but we can check the environment. Put the following after the rest of the CSP configs and it should work.

```python
if config('SWITCH_FLASHTALKING', default=DEV, cast=bool):
    CSP_IMG_SRC += ('servedby.flashtalking.com',)
```